tensorflow-notebook/Dockerfile

```diff
@@ -7,8 +7,7 @@ LABEL maintainer="Jupyter Project <jupyter@googlegroups.com>"
 
 # Install Tensorflow
 RUN mamba install --quiet --yes \
-    'tensorflow=2.4.1' \
-    && \
```

Comment:
@mathbunnyru is it the style you want to use? If it's the case I have fixed it.

Reply:
No, I added the comment about the line I didn't like.

Reply:
Ok because I've rolled back your previous style change and I made the link with your other remark on style with `&&` at the beginning of a line. So I would like to be sure.

Reply:
Thanks, right now the style looks good and consistent :)